### ISSUE TITLE
Concept mentions to use `_term_ (x.y)` format

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-bipm#245]
Right now in BIPM documents we render concept mentions like this:

* "... _{term}_ [term defined in Section 2.2] ..."

<img width="633" alt="Screen Shot 2023-02-07 at 11 44 55 AM" src="https://user-images.githubusercontent.com/11865/217143149-0df2fea1-4e3f-4198-8bc5-270da0226ab6.png">

However, it would look much nicer to have this:

* "... _{term}_ (2.2) ..."

**Comment by opoudjis** [comment]
I had that default set in isodoc, to be overridden only in ISO, since it was clearer to understand. But it is too wordy, so I'll get rid of it, and make it just a cross-reference.

The cross-reference will of course obey the format of the gem, which in almost all flavours will mean that "Clause" will precede it, as it should.
